jlm/llvm/opt/alias-analyses/Andersen.cpp

```diff
@@ -898,6 +795,18 @@ Andersen::AnalyzeRvsdg(const rvsdg::graph & graph)
   }
 }
 
+void
+Andersen::SetConfiguration(PassConfiguration config)
+{
+  Config_ = config;
```

Comment:
`std::move(config)`. It does not make much of a difference now, but might going forward.